### ISSUE TITLE
[FastPitch/Tensorboard] The line of training loss and validation loss is not common

### ISSUE THREAD

**Issue by NoahDrisort** [NVIDIA/DeepLearningExamples#608]
![image](https://user-images.githubusercontent.com/36092323/87551894-2d65b880-c6db-11ea-8f4d-c459a8032cbf.png)

When I train model FastPitch from NVIDIA source code, I have the images like above,
My train data is 11239 and validation is 28,
I have seen that the train line and val line more and more separate it others. It seems not common
I feel so confused?
Thank you for sharing the code <3

**Comment by Truong-Thanh-Quang** [comment]
I have the same issue.
![image](https://user-images.githubusercontent.com/20198693/91669945-369fcd00-eb43-11ea-8af1-c5c4765a41ba.png)

Commandline output:
![image](https://user-images.githubusercontent.com/20198693/91669968-6058f400-eb43-11ea-9a84-9c9a1ea1bbe8.png)

**Comment by NoahDrisort** [comment]
> I have the same issue.
> ![image](https://user-images.githubusercontent.com/20198693/91669945-369fcd00-eb43-11ea-8af1-c5c4765a41ba.png)
> 
> Commandline output:
> ![image](https://user-images.githubusercontent.com/20198693/91669968-6058f400-eb43-11ea-9a84-9c9a1ea1bbe8.png)
I wonder if FastPitch work well in Vietnamese, or it may depend on pretrain Tacotron2 quality

**Comment by Truong-Thanh-Quang** [comment]
> > I have the same issue.
> > ![image](https://user-images.githubusercontent.com/20198693/91669945-369fcd00-eb43-11ea-8af1-c5c4765a41ba.png)
> > Commandline output:
> > ![image](https://user-images.githubusercontent.com/20198693/91669968-6058f400-eb43-11ea-9a84-9c9a1ea1bbe8.png)
> > I wonder if FastPitch work well in Vietnamese, or it may depend on pretrain Tacotron2 quality

The pretrain is very good. The inference likely the original one.

**Comment by NoahDrisort** [comment]
Have you try to set pitch loss and duration loss equal to zero

**Comment by Truong-Thanh-Quang** [comment]
I have not set pitch loss and duration loss equal to zero, yet. I dont know about that.
The inference from FastPitch sounds better than pretrain tacotron2. I think the result is OK but it is a little strange on validation line.

**Comment by NoahDrisort** [comment]
This repo has the same problem: https://github.com/ming024/FastSpeech2, 
They said that their validation is very small, they validate model base on train loss
What are your validation size and other loss have the same shape like dur-error above?

**Comment by alancucki** [comment]
Hi @NoahDrisort @Truong-Thanh-Quang 

sorry for a late reply. It's natural for the training and validation curves to be so wide apart. The main reason is the temporal disagreement - it's L2 loss, and we try to match the spectrograms exactly. Even a slight shift in time will cause a mismatch.

Instead of monitoring validation loss, I'd advise to implement more sophisticatd metrics that take into account the temporal disagreement, or listen to the samples and perform pairwise comparisons (see 4.1.1. in the [paper](https://arxiv.org/abs/2006.06873))
